### PR TITLE
Allow IV to be specified in the "key_info_string"

### PR DESCRIPTION
**Pull request by jforaci**
Allow an optional initialization vector to be specified as `iv` in the "key_info_string" for raw key encryption. This is for use in conjunction with FairPlay where the IV will be delivered from the KSM, not the randomized one otherwise supplied in the ISO BMFF `tenc` box. The --iv option is mutually exclusive with this key parameter.

### CODE REVIEW

**Comment by jforaci** [comment]
@googlebot I signed it!

File: docs/source/options/raw_key_encryption_options.rst [hunk 1]
```diff
@@ -27,6 +27,10 @@ Raw key encryption options
 
     *key_id* and *key* should be 32-digit hex strings.
 
+    *initialization_vector* is an optional IV with the same format and semantics
+    as the parameter for the --iv option below. This is mutually exclusive with
```
Comment (kqyang) [inline docs/source/options/raw_key_encryption_options.rst:31]: enclose "--iv" with "*"

File: docs/source/tutorials/raw_key.rst [hunk 2]
```diff
@@ -20,7 +20,7 @@ Synopsis
 
 **key_info_string** is of the form::
 
-    label=<label>:key_id=<key_id>:key=<key>
+    label=<label>:key_id=<key_id>:key=<key>[:iv=<initialization_vector>]
```
Comment (kqyang) [inline docs/source/tutorials/raw_key.rst:23]: Can you update the one of the examples below to include the new field, e.g. on "Example with FairPlay using 'cbcs' protection scheme::"?

File: packager/app/packager_main.cc [hunk 3]
```diff
@@ -207,6 +208,17 @@ bool ParseKeys(const std::string& keys, RawKeyParams* raw_key) {
                  << value_map[kKeyLabel];
       return false;
     }
+    if (!value_map[kKeyIvLabel].empty()) {
+      if (!raw_key->iv.empty()) {
+        LOG(ERROR) << "IV already specified with --iv";
+        return false;
+      }
+      if (!base::HexStringToBytes(value_map[kKeyIvLabel], &key_info.iv)) {
+        LOG(ERROR) << "Empty key or invalid hex string for IV: "
```
Comment (kqyang) [inline packager/app/packager_main.cc:217]: replace "key" with "IV"

**Comment by jforaci** [comment]
@kqyang I made the suggested changes, thank you!

**Pull request merged**